examples/distributed/README.md

```diff
@@ -0,0 +1,63 @@
+# Launch distributed DeepGNN in cluster
+
+## Using ansible
+
+### Preparation
+
+1. Linux nodes
+
+    a) Setup SSH: https://docs.microsoft.com/en-us/azure/virtual-machines/linux/ssh-from-windows
+
+    b) Go to `Inventory` folder and add your linux servers in `linux` section:
+
+    ```YAML
+    linux:
+      hosts:
+        azure-linux-vm01: # machine alias
+          ansible_host: x.x.x.x # machine ip
+      vars:
+        ansible_connection: paramiko
+        ansible_user: azureuser # ssh user
+        ansible_ssh_pass: "password" # ssh password
+    ```
+
+2. Windows nodes
```

Comment:
where is a CI test for windows?

Reply:
I did some investigation about how to do that in github action because ansible doesn't support windows control node. If we want to test it on windows, we need to setup 2 agents, one for linux to run the ansible script and 1 for windows to run the GE. I didn't find a good solution yet because it is a little bit complicated to orchestrate these 2 nodes in github action. We need to make sure these two agents are in same vnet or a public dns which is used to find each other...

Do you have some suggestions? Thanks!